.github/workflows/docs.yml

```diff
@@ -6,6 +6,8 @@ on:
       - "docs/**"
     branches:
       - "**"
+    pr:
+      - "docs/**"
```

Comment:
I’m not entirely sure this is working as expected. I tried [opening a PR from a fork](https://github.com/alephdata/aleph/pull/3703) but the workflow isn’t triggered.

I think maybe all that’s required is to switch from the `push` trigger to the `pull_request` trigger?

```yml
on:
  pull_request:
    paths:
      - "docs/**"
  workflow_dispatch: {}
```

If I understand the docs correctly, this should trigger the workflow whenever a pull request is opened, updated etc. and includes changes in the `docs` subdirectory.

In addition, we could add a trigger to also run the workflow after changes have been merged into the `main` or `develop` branch:


```yml
on:
  pull_request:
    paths:
      - "docs/**"
  push:
    paths:
      - "docs/**"
    branches:
      - "develop"
      - "main"
  workflow_dispatch: {}
```